.github/workflows/test-and-publish.yaml

```diff
@@ -274,6 +272,19 @@ jobs:
         with:
           name: wheel-${{ github.run_id }}-${{ github.sha }}
           path: ./dist/
+  check-install-from-sdist:
```

Comment:
I included this change, which just downloads the sdist and tries to install it on ubuntu-24.04